.circleci/config.yml

```diff
@@ -132,6 +135,7 @@ jobs:
       - image: circleci/golang:1.12
```

Comment:
would this make sense if updated to golang1.13 too as in `Dockerfile`

Reply:
Good call!